views/homepage.erb

```diff
@@ -4,12 +4,16 @@
   <link href="<%= url('/styles.css') %>" rel="stylesheet" type="text/css" />
```

Comment:
I think linking the .css file would work even without including it in the url()? So if we write href="/styles.css", it technically should work.

Reply:
Yes, it makes sense, and it works perfectly, thank you!